packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/App.xaml.cs

```diff
@@ -53,9 +53,8 @@ public App()
         protected override void OnLaunched(LaunchActivatedEventArgs e)
         {
             base.OnLaunched(e);
-            var frame = Window.Current.Content as Frame;
-            frame.Navigate(typeof(MainPage));
-            Window.Current.Activate();
+            var frame = (Frame)Window.Current.Content;
```

Comment:
>(Frame)Window.Current.Content; [](start = 24, length = 30)

Note that in C# the 'as' conversion returns null if conversion failed, while the C-style conversion throws an exception. It is different from the C++/WinRT where the 'as' conversion throws an exception while 'try_as' returns null in case of failure.
This is why the code here is changed here to use C-style conversion to match the C++/WinRT 'as()' function call which is a correct functionality since we do not check for null after the call.